build-logic/src/main/kotlin/me.champeau.convention-testing.gradle.kts

```diff
@@ -5,7 +5,7 @@ plugins {
 val wrapperVersion: String = GradleVersion.current().version
 
 // The plugin is broken with Gradle 5.6.*
-val otherVersions = listOf("6.9.1", "7.2", "7.3", "7.4")
+val otherVersions = listOf("7.6.2")
```

Comment:
Gradle 7.6 is the first version that [supports Java 19](https://docs.gradle.org/7.6/release-notes.html#java19).